CodeEdit/Features/Documents/Controllers/CodeEditDocumentController.swift

```diff
@@ -37,6 +37,10 @@ final class CodeEditDocumentController: NSDocumentController {
         return panel.url
     }
 
+    override func noteNewRecentDocument(_ document: NSDocument) {
+        // The super method is run manually when opening new documents.
```

Comment:
Manually or automatically?

Reply:
The super method is called a few lines below, that's what I meant with "manually". When a new document is added to the windowcontroller, this fucntion  is called automatically. Now, it doesn't call the super method anymore.

Reply:
Ah okay, so it overridden because we don't want to run the default implementation?

Reply:
Correct!